mmf/utils/checkpoint.py

```diff
@@ -321,11 +321,12 @@ def _load_pretrained(self, ckpt):
             key += "."
             value += "."
             for attr in ckpt:
+                if hasattr(model, "format_state_key"):
+                    formatted_attr = model.format_state_key(attr)
+                else:
+                    formatted_attr = attr
+
```

Comment:
Rebase on master.

Reply:
this is a minor refactor to move the formatted_attr assignment to the outer loop since it does not need to be computed every time.